pkg/metrics/metrics.go

```diff
@@ -0,0 +1,39 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// HyperConvergedPodsCounter for pods hyper-converged by stork scheduler i.e scheduled on
+	// node where replicas for all pod volumes exists
+	HyperConvergedPodsCounter = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
These should be in the extender and health monitor packages. Just have helper methods here to register the metrics